README.md

```diff
@@ -23,7 +23,7 @@ Create a `Clock` component updating the time every second.
 1. Change the `Clock` component to print the time not only on the page but also in the `DevTools` console.
```

Comment:
Need to add console log printing each tick of clock 
add it and can be reviewed by mentors